AntennaTracker/Parameters.h

```diff
@@ -130,7 +134,8 @@ class Parameters {
     AP_Float yaw_slew_time;
     AP_Float pitch_slew_time;
     AP_Float min_reverse_time;
-    AP_Float scan_speed;
+    AP_Float scan_speed_yaw;
```

Comment:
scanning slower than 1deg/dec is pretty darn slow. Might as well take the opportunity to make this int8_t or int16_t. The param description says increment by 1 which implies integer. May need to cast to float where it gets used.